### PR TITLE
Fix incomplete GraphQL aggregations (required numeric fields)

### LINKED ISSUE #2697: GraphQL aggregation incomplete with required numeric fields
**Informations**
- **Node.js version**: 10.8.0
- **NPM version**: 6.6.0
- **Strapi version**: 3.0.0-alpha.21
- **Database**: Mongo DB 4.0.4
- **Operating system**: Linux (Ubuntu)
- **(Optional) Link to your Project**: -

**What is the current behavior?**
Required fields do not have sum/avg/min/max added to the generated aggregator types in the GraphQL plugin.

**Steps to reproduce the problem**
1. Setup Strapi with the GraphQL plugin enabled
2. Add a content type with a number field that is required.
3. Go to localhost:1337/graphql and open the (green) schema button
4. Navigate to the aggregator for the content type we just created and it has only the count attribute 

**What is the expected behavior?**
Every numeric type (required or not) should be part of the aggregate.

**Suggested solutions**
As a workaround, I disabled the field from being required. Actual solution would be in plugins/graphql/services/Aggregator.js:454, where the numeric fields are extracted, the main error is caused by the function isNumericType on line 60. If the field is required, the type would be Int! or Float!, which will fail the isNumberType check. The code below would fix this.
```js
  isNumberType: type => {
    if (type.endsWith('!')) {
      type = type.slice(0, -1);
    }

    return type === 'Int' || type === 'Float';
  },
```

### PR DESCRIPTION
**Pull request by Convly**
fix #2697 

**What did you do?**
Removed the required marker when checking for number types in (int/float) **strapi-plugin-graphql**

### CODE REVIEW

File: packages/strapi-plugin-graphql/services/build-aggregation.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const { toSDL } = require('./schema-definitions');
  * @returns {Boolean}
  */
 const isPrimitiveType = _type => {
-  const type = _type.replace('!', '');
+  const type = noRequired(_type);
```
Comment (alexandrebodin) [inline packages/strapi-plugin-graphql/services/build-aggregation.js]: Can you rename the vars so it is type and nonRequiredType just to make more sense

File: packages/strapi-plugin-graphql/services/build-aggregation.js [hunk 2]
```diff
@@ -59,7 +59,8 @@ const isNotOfTypeArray = type => {
 /**
  * Returns all fields of type Integer or float
  */
-const isNumberType = type => {
+const isNumberType = _type => {
+  const type = noRequired(_type);
```
Comment (alexandrebodin) [inline packages/strapi-plugin-graphql/services/build-aggregation.js]: same here :)

Comment (Convly) [reply packages/strapi-plugin-graphql/services/build-aggregation.js]: 032d3ba83d321900ef5132234210c157f14f45ac

File: packages/strapi-plugin-graphql/services/build-aggregation.js (hunk 1 above)
Comment (Convly) [reply packages/strapi-plugin-graphql/services/build-aggregation.js]: 032d3ba83d321900ef5132234210c157f14f45ac

**Pull request merged**